recipes/sparrow/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "sparrow" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xtensor-stack/sparrow/archive/{{ version }}.tar.gz
+  sha256: 72ee4838388666eda4b557c5340af40d4f0c7d1d230ea230e256e4df1e0cab13
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/sparrow/buffer.hpp                                           # [unix]
+    - test -f ${PREFIX}/include/sparrow/sparrow_version.hpp                                  # [unix]
+    - test -f ${PREFIX}/share/cmake/sparrow/sparrowConfig.cmake                              # [unix]
+    - test -f ${PREFIX}/share/cmake/sparrow/sparrowConfigVersion.cmake                       # [unix]
+    - test -f ${PREFIX}/share/cmake/sparrow/sparrowTargets.cmake                             # [unix]
+    - if not exist %LIBRARY_PREFIX%\include\sparrow\buffer.hpp (exit 1)                      # [win]
+    - if not exist %LIBRARY_PREFIX%\include\sparrow\sparrow_version.hpp (exit 1)             # [win]
+    - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowConfig.cmake (exit 1)         # [win]
+    - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowConfigVersion.cmake (exit 1)  # [win]
+    - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowTargets.cmake (exit 1)        # [win]
+
+about:
+  summary: C++20 idiomatic APIs for the Apache Arrow Columnar Format
+  description: C++20 idiomatic APIs for the Apache Arrow Columnar Format
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  home: https://github.com/xtensor-stack/sparrow
+  dev_url: https://github.com/xtensor-stack/sparrow
+
+extra:
+  recipe-maintainers:
+    - jjerphan
```

@JohanMabille and @Klaim: do you consent maintaining the feedstock?
```suggestion
    - jjerphan
    - JohanMabille
    - Klaim
```

---

👍🏽

---

Yes!